pkg/object/httpserver/mux.go

```diff
@@ -253,6 +284,7 @@ func (mi *muxInstance) serveHTTP(stdw http.ResponseWriter, stdr *http.Request) {
 				StatusCode: statusCode,
 				ReqSize:    uint64(reqMetaSize) + uint64(body.BytesRead()),
 				RespSize:   respSize,
+				RespHeader: respHeader,
```

Comment:
it seems `RespHeader` is added to `httpstat.Metric` only for carrying the response header to the logger, then it should not be part of `httpstat.Metric`, we can use a local variable to hold the header.